manifests/yamls/nats-server.yaml

```diff
@@ -1,11 +1,320 @@
 ---
-# Source: nats-server/templates/nats.yaml
-apiVersion: "nats.io/v1alpha2"
-kind: "NatsCluster"
+# Source: nats/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: "devtron-nats"
-  namespace: devtroncd
+  name: devtron-nats-config
```

Comment:
latest nats has better monitoring support and dashboards please add it on roadmap

Reply:
Done